### DISCUSSION TITLE
help wanted

### DISCUSSION THREAD

**Discussion by achiragaming** [lavalink-devs/Lavalink#896] [Q&A] [answered]
hey im asking for a help this is not a bug or anything

is there anyway to get the current playing url to radio url like with all events

**Comment by topi314** [comment, marked as answer]
are you implementing your own client? If not this should be a question for your client lib maintainers.

Lavalink offers an extensive REST API since v3.7
See https://github.com/lavalink-devs/Lavalink/blob/master/IMPLEMENTATION.md#get-player for a request to get the current player of a guild which includes the current playing track
